docs/semantics-of-k.tex

```diff
@@ -6350,7 +6350,7 @@ \subsubsection{Wellformedness}
 \item In any pattern that is a head being applied to a list of argument 
 patterns, the number and sorts of argument patterns should match with the head 
 declaration;
-\item Every sorts, symbols, and aliases should be declared to be used;
+\item All sorts, symbols, and aliases should be declared before used;
```

Comment:
"before being used", "before they are used", "before usage" or something else instead of "before used" (used is a participle or an adjective so it should be composed with something before attaching "before" to it).